client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/db/HeuristicsCalculator.java

```diff
@@ -490,11 +499,35 @@ private Object getValue(Expression exp, DataRow data) {
                 }
             }
         } else if (exp instanceof CastExpression) {
-            CastExpression castExpression =(CastExpression)exp;
+            CastExpression castExpression = (CastExpression) exp;
             return getValue(castExpression.getLeftExpression(), data);
+        } else if (exp instanceof DateTimeLiteralExpression) {
+            DateTimeLiteralExpression dateTimeLiteralExpression = (DateTimeLiteralExpression) exp;
+            String str = dateTimeLiteralExpression.getValue();
+            assert (str.length() > 2 && startsAndEndsWithQuotes(str));
+            str = removeQuotes(str);
+            return str;
         } else {
             cannotHandle(exp);
             return null;
         }
     }
+
+    /**
+     * Given a string "Hello World" returns "ello Worl".
```

Comment:
this description is confusing... so it is not `removingQuotes` but always removing first and last char? if so, should change name of function (eg, `removeFirstAndLast`), or change semantics (ie, throw an execption if first and last are not quotes). also should it deal only with `"` or also `'`?

Reply:
Changed to removeFirstAndLastCharacter()